src/main/java/io/confluent/kafka/connect/datagen/DatagenTask.java

```diff
@@ -154,16 +159,23 @@ public void start(Map<String, String> props) {
       } catch (IllegalArgumentException e) {
         log.warn("Quickstart '{}' not found: ", quickstartName, e);
       }
+    } else if (schemaString != "") {
+      generator = generatorBuilder.schemaString(schemaString).build();
     } else {
+      String err = "Unable to read the '" + schemaFilename + "' schema file";
       try {
-        generator = new Generator.Builder()
-            .schemaStream(new FileInputStream(schemaFilename))
-            .random(random)
-            .generation(count)
-            .build();
+        generator = generatorBuilder.schemaStream(new FileInputStream(schemaFilename)).build();
+      } catch (FileNotFoundException e) {
+        // also look in jars on the classpath
+        try {
+          generator = generatorBuilder
```

Comment:
I added this because the tests started failing once I removed the `quickstart` property when `schema.file` was set. Previously they were passing because the connector was just running using the `quickstart` property (since both `quickstart` and `schema.file` were set). However even though I added it just for tests, it seems like a reasonable extension to the property (look in the jars as well as the filesystem).